google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -78,13 +77,13 @@
   private JLabel noDockerfileLabel;
   private JLabel errorIcon;
   private JLabel errorMessage;
-  private AppEngineDeploymentConfiguration deploymentConfiguration;
+  private AppEngineFlexibleFacetConfiguration facetConfiguration;
   private AppEngineHelper appEngineHelper;
 
-  FlexibleFacetEditor(@Nullable AppEngineDeploymentConfiguration deploymentConfiguration,
+  FlexibleFacetEditor(AppEngineFlexibleFacetConfiguration facetConfiguration,
```

Comment:
I don't think either is default. Having these just aids in static analysis. I.e. the IDE will give useful warnings if you pass null to an @NotNull argument.